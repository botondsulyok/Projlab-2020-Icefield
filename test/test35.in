field stable 0
field unstable 2 false 0
neighbours stable unstable
neighbours unstable stable
entity esk Eskimo unstable
entity pb icebear stable
item ds1 suit esk
ability unstable esk
stats stable
stats unstable
stats esk
stats pb
move pb unstable
stats stable
stats unstable
stats esk
stats pb