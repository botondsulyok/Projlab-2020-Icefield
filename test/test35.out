id: stable
type: iceunit
building: plains0
snow: 0
pickable: null
standers: pb
pullers: null
masters: null
neighbours: unstable

id: unstable
type: unstable
building: igloo0
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: stable
capacity: 2
broken: false

id: esk
type: eskimo
iceUnit: unstable
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: ds1
pullerDevice: basicpull0
tent: null
components: null

id: pb
type: icebear
iceUnit: stable
movebehaviour: amph0

Game Over
id: stable
type: iceunit
building: plains0
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: unstable

id: unstable
type: unstable
building: plains3
snow: 0
pickable: null
standers: esk, pb
pullers: esk
masters: esk
neighbours: stable
capacity: 2
broken: true

id: esk
type: eskimo
iceUnit: unstable
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: ds1
pullerDevice: basicpull0
tent: null
components: null

id: pb
type: icebear
iceUnit: unstable
movebehaviour: amph0

